src/renderer/rendererutils.ts

```diff
@@ -1,24 +1,20 @@
-import { instanceDifficulty, InstanceDifficultyType } from "main/constants";
+import { instanceDifficulty, InstanceDifficultyType, VideoCategory } from "main/constants";
 
 /**
  * isCategoryPVP
  */
- const isCategoryPVP = (category: string) => {
+ const isCategoryPVP = (category: VideoCategory) => {
     if (!category) return false;
 
     const pvpCategories = [
-        "2v2", 
-        "3v3", 
-        "Skirmish", 
-        "Solo Shuffle", 
-        "Battlegrounds"
+        VideoCategory.TwoVTwo,
+        VideoCategory.ThreeVThree,
+        VideoCategory.Skirmish,
+        VideoCategory.SoloShuffle,
+        VideoCategory.Battlegrounds,
     ];
 
-    if (pvpCategories.includes(category)) {
-        return true;
-    } else {
-        return false;
-    }
+    return pvpCategories.indexOf(category) !== -1;
```

Comment:
Why not just `.includes()`?

Reply:
My bad. No idea why I thought that was a good idea to change.